website/sidebars.js

```diff
@@ -2,10 +2,14 @@ const sidebarSettings = {
   docs: [
     "docs/introduction",
     {
-      type: "link",
-      label: "Guides",
-      href: `/guides`,
-    },
+      type: "category",
+      label: "Get started with dbt",
```

Comment:
Unless we have plans on the table to add content for Core, it should probably look like this:
```suggestion
      label: "Get started with dbt Cloud",
```

Reply:
oh we do have core content here: https://github.com/dbt-labs/docs.getdbt.com/pull/5269/files#diff-202ac695f0bff05c7176aaa16d547ff98838402c105e3ab1446420db5b439086R61

is that what you mean?